colin-api/src/colin_api/models/filing.py

```diff
@@ -1996,7 +1996,7 @@ def add_correction_filings(cls, con, filing: Filing) -> list:
                                       'filing_type': filing.filing_type,
                                       'filing_sub_type': None})
 
-            if not filings_added:  # only comment added
+            if bool(filing.body.get('commentOnly', False)):  # only comment added
```

Comment:
Add this check in (line 2006) before returning filings_added
```suggestion

if not filings_added:  # if no filing created
                 raise GenericException(  # pylint: disable=broad-exception-raised
                     f'No filing created for this correction identifier:{corp_num}.',
                     HTTPStatus.NOT_IMPLEMENTED
                 )
```